src/Components/ArticleCard.jsx

```diff
@@ -2,14 +2,19 @@ import { Link } from "react-router-dom";
 import "./ArticleCard.css";
 
 export default function ArticleCard({ article }) {
+  const articleDate = article.created_at
+    .slice(0, 10)
+    .split("-")
+    .reverse()
+    .join("-");
```

Comment:
If you're using this to format dates in multiple places, I would consider moving this out into a util function so you can just invoke it with article.created_at whenever you need to.